### PR TITLE
Validate facility codes against master list

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@bruskiza hello from Uganda! Please review & merge when ready.
camera_2015_09_11__12_21_pm

File: test/app.test.js [hunk 1]
```diff
@@ -15,7 +16,10 @@ describe("app", function() {
 
             tester
                 .setup.config.app({
-                    name: 'test_app'
+                    name: 'test_app',
+                    facility_codes: JSON.parse(
+                        fs.readFileSync(
+                            "src/lookups/facility_codes.json", "utf8"))
```
Comment (bruskiza) [inline test/app.test.js:22]: I can see this for testing... is production just read out of the config?

**Comment by bruskiza** [comment]
:+1:

File: test/app.test.js (hunk 1 above)
Comment (smn) [inline test/app.test.js:22]: Yes, for now? At a later stage we'd probably stick this in an API?

**Pull request merged**